src/openms/source/FORMAT/HANDLERS/MzXMLHandler.cpp

```diff
@@ -988,9 +1039,19 @@ namespace OpenMS
         }
       }
 
-      os << "\t</msRun>\n"
-         << "\t<indexOffset>0</indexOffset>\n"
-         << "</mzXML>\n";
+      os << "\t</msRun>\n";
+
+      // create scan index (does not take a lot of space and is required for MaxQuant)
```

Comment:
I'd rename the FileConverter flag to `-write_index` then, since it applies to both mzML and mzXML now (after #2521 is merged)
Will adapt the docs accordingly.